sw/device/silicon_creator/rom/BUILD

```diff
@@ -318,3 +319,12 @@ pkg_files(
     srcs = [":pre_package"],
     prefix = "earlgrey/rom",
 )
+
+manual_test(
+    name = "manual",
```

Comment:
nit: this should be named `manual_test`.

Reply:
done